docs/self-managed/operate-deployment/operate-authentication.md

```diff
@@ -137,29 +142,43 @@ export SPRING_PROFILES_ACTIVE=identity-auth
 
 Identity requires the following parameters:
 
-| Parameter name                                      | Description                                                                                                                                   | Example value                                                                     |
+:::tip
+Using Microsoft Entra ID (Azure AD)?
+See [Connecting to Entra ID](../identity/user-guide/configuration/connecting-to-entra-id.md).
+:::
+
+| Environment variable                | Description                               | Example value                                       |
+| ----------------------------------- | ----------------------------------------- | --------------------------------------------------- |
+| CAMUNDA_IDENTITY_TYPE               | Type of Identity provider.                | AZUREAD (defaults to KEYCLOAK)                      |
+| CAMUNDA_IDENTITY_BASE_URL           | URL of the Identity component             | http://localhost:8080                               |
+| CAMUNDA_IDENTITY_ISSUER             | URL of issuer (Identity)                  | http://localhost:18080/auth/realms/camunda-platform |
+| CAMUNDA_IDENTITY_ISSUER_BACKEND_URL | Backend URL of issuer (Identity)          | http://localhost:18080/auth/realms/camunda-platform |
+| CAMUNDA_IDENTITY_CLIENT_ID          | Similar to a username for the application | operate                                             |
+| CAMUNDA_IDENTITY_CLIENT_SECRET      | Similar to a password for the application | XALaRPl...s7dL7                                     |
+| CAMUNDA_IDENTITY_AUDIENCE           | Audience for Operate                      | operate-api                                         |
+
```

Comment:
I would actually also include this part in the other document. In the end all of the configuration variables will be the same for most components (probably except Web-Modeler and Connectors?), so Having it in a central place would reduce duplication. Here we could link to the `connecting-to-entra-id.md` page

Reply:
I like this suggestion, I'll refactor